code/datums/armor/_armor.dm

```diff
@@ -0,0 +1,221 @@
+GLOBAL_LIST_INIT(armor_by_type, __generate_armor_cache())
+
+/proc/__generate_armor_cache()
+	. = list()
```

Comment:
You don't need this, nothing in your proc errors, but if you want to keep it, `return .` at the end